webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -9,33 +9,42 @@ import {
   notificationsDrawer as sessionStorage
 } from '../../../common/sessionStorage';
 import API from '../../../API';
-const getNotificationsInterval = 10000;
+import { isNil } from 'lodash';
+const defaultNotificationsPollingInterval = 10000;
+const notificationsInterval = isNil(process.env.NOTIFICATIONS_POLLING) ?
+  defaultNotificationsPollingInterval :
+  process.env.NOTIFICATIONS_POLLING;
```

Comment:
yes, but not here. in webpack.config.js... and there ther require('dotenv') line should probably come before the DefinePlugin (where NOTIFICATIONS_POLLING is defined)